install/vep_cache_grch37.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Installs cache data for ensembl-vep
+
+SPECIES="homo_sapiens"
+ASSEMBLY="GRCh37"
```

Comment:
so this will only work for GRCh37?

Reply:
Right... For now, we are only planning on using it for ichange. Due to the constraints of the cache, it seemed preferable to set this limitation. But there is the upcoming renata deployment for which GRCh38 is hinted...
In that case the variable should be changed to the proper value, but the toplevel cache file should be updated as well (with GRCh37 it hasn't been updated since version 75)

In a related note, it's unclear at the moment wether the current Bento-Web for example can really handle GRCh38 as it pretends to (some things have been hard coded)...